dbt/include/greenplum/macros/adapters.sql

```diff
@@ -122,6 +122,21 @@
   ({{ comma_separated_columns }});
 {%- endmacro %}
 
+{% macro greenplum__generate_schema_name(custom_schema_name, node) -%}
```

Comment:
Hi! Could you explain the reason for this macro? I don't see this macro here https://docs.getdbt.com/docs/contributing/building-a-new-adapter
If you want to specify schema, you can write something like that:
```
{{
    config(
        schema='<schema_name>',
    )
}}
```

Reply:
Native dbt-function would compile target schema name as a compilation `{profile.schema}_{project.schema}`
That means that every custom schema should has schema name from your profile.yml

For example if I have `schema: public` in profiles.yml
And than I specify schema in model's config, let's say `config(schema='my_data_marts')`
Compiled SQL will be `select * from public_my_data_marts.table_name`

I don't know if someone uses this as naming convension.

Reply:
I just dropped the `{profile.schema}_` in the `{profile.schema}_{project.schema}`

So that compiled sql from my example would be `select * from my_data_marts.table_name`

Reply:
You are right! We are dealing with it by setting up some prefix in our profile.yml. Thank you)

Reply:
We have over 600+ sources - would be overkill to store 600+ similar profiles)

Reply:
Haha

I've updated package on pypi https://pypi.org/project/dbt-greenplum/1.0.4.1/